### PR TITLE
Fix #142875 www.imdb.com

### LINKED ISSUE #142875: www.imdb.com
### Issue URL (Annoyance)
 [https://www.imdb.com/title/tt8097030/](https://adguardteam.github.io/AnonymousRedirect/redirect.html?url=https%3A%2F%2Fwww.imdb.com%2Ftitle%2Ftt8097030%2F)
### Comment
Username: `@rakleed`
> 

### Screenshots
<details>
  <summary>Screenshot 1</summary>

  Screenshot 1
</details>

### System configuration

Information | value
--- | ---
AdGuard product: | AdGuard for Mac v2.9.2.1234+release
Browser: | Safari
Stealth mode options: | Hide your search queries, <br>Remove X-Client-Data header from HTTP requests, <br>Strip URLs from tracking parameters, <br>Disable cache for third-party requests, <br>Hide your Referrer from third-parties (), <br>Hide your User-Agent ()
DNS filtering: | server: `https://dns.cloudflare.com/dns-query`<br>
Filters: | <b>Ad Blocking:</b><br/>AdGuard Base<br/><br/><b>Privacy:</b><br/>AdGuard Tracking Protection, <br/>AdGuard URL Tracking<br/><br/><b>Social Widgets:</b><br/>AdGuard Social Media<br/><br/><b>Annoyances:</b><br/>AdGuard Annoyances<br/><br/><b>Language-specific:</b><br/>AdGuard Russian
Custom filters: | https://gitlab.com/magnolia1234/bypass-paywalls-clean-filters/-/raw/main/bpc-paywall-filter.txt
Userscripts: | IMDb+Tomatoes+(url:+https://greasyfork.org/scripts/15222-imdb-tomatoes/code/IMDb%20Tomatoes.user.js),<br>AdGuard+Extra+(url:+https://userscripts.adtidy.org/release/adguard-extra/1.0/adguard-extra.user.js),<br>Bypass+Paywalls+Clean+-+en+(url:+https://gitlab.com/magnolia1234/bypass-paywalls-clean-filters/-/raw/main/userscript/bpc.en.user.js),<br>Return+YouTube+Dislike+(url:+https://github.com/Anarios/return-youtube-dislike/raw/main/Extensions/UserScript/Return%20Youtube%20Dislike.user.js)

### PR DESCRIPTION
**Pull request by rakleed**
https://github.com/AdguardTeam/AdguardFilters/issues/142875

Now only pop-up is blocked, which automatically appears (without a user request). That is, I solved the problems that were reported in #141159

### CODE REVIEW

**Comment by rakleed** [comment]
@zloyden

File: AnnoyancesFilter/Popups/sections/popups_specific.txt [hunk 1]
```diff
@@ -7,6 +7,8 @@
 !
 ! SECTION: Popups - Regular rules
 !
+imdb.com#$?#div[data-testid="promptable"]:has(.ipc-rating-display-with-poster__image) { display: none !important; }
```
Comment (krystian3w) [inline AnnoyancesFilter/Popups/sections/popups_specific.txt:10]: I think coreLibs should correctly exclude this from `content_blocker` Filter version.

```suggestion
imdb.com#?#div[data-testid="promptable"]:has(.ipc-rating-display-with-poster__image)
```

Comment (zloyden) [reply AnnoyancesFilter/Popups/sections/popups_specific.txt:10]: ` { display: none !important; }` is not necessary for `#?#` rules - they won't apply on content blockers. So, this part of the rule should be removed.
@AdamWr @BlazDT could you check it on your side? I don't have the popup.

Comment (AdamWr) [reply AnnoyancesFilter/Popups/sections/popups_specific.txt:10]: I cannot reproduce it.

Regarding second rule:
```adblock
imdb.com#$?#body:has(div[data-testid="promptable"]:has(.ipc-rating-display-with-poster__image)) { overflow: auto !important; }
```

If I'm not wrong, it's not necessary to use `:has` two times, something like this:
```adblock
imdb.com#$?#body:has(div[data-testid="promptable"] .ipc-rating-display-with-poster__image) { overflow: auto !important; }
```
should match the same element.

Comment (rakleed) [reply AnnoyancesFilter/Popups/sections/popups_specific.txt]: Fixed.

**Comment by rakleed** [comment]
@zloyden @AdamWr to reproduce, follow the link https://hostigram.xyz/?tkn=ZHPyrVjxQXxsl7vifxMYNoVyA1p4yLB9F6NrNYzAW5IwT&dl=1 (and click Download if necessary) to view the saved page. Don't forget to add the `hostigram.xyz` domain to the rules.

**Review by zloyden — COMMENTED** [review]
https://github.com/AdguardTeam/AdguardFilters/pull/142876#issuecomment-1431271287 I've seen that the rules work there, but it's better to test them on `imdb.com`.
The only option I can suggest is to move the rules to AdGuard Experimental and test them for some time.

**Comment by rakleed** [comment]
In general, if you are logged into IMDb and go to the page of one movie / series every couple of hours, then this window appears at 4-5 opening, which annoys me as an active user of the site.

@zloyden done.

**Review by zloyden — APPROVED** [review]

**Pull request merged**